jax/_src/numpy/lax_numpy.py

```diff
@@ -100,9 +100,6 @@ def canonicalize_shape(shape: Any, context: str="") -> core.Shape:
 e = np.e
 euler_gamma = np.euler_gamma
 inf = np.inf
-NINF = np.NINF
```

Comment:
Let's leave these in for now with a `# TODO: reason` comment to remove them.

You could change them to:
```
NINF = -np.inf
```
to fix the NumPy issue.

The reason is that these are reexporting NumPy's constants as part of JAX's API, and to change JAX's API we should follow a deprecation process.

Reply:
Sure! I moved them back.

Reply:
It's fine to change these to be *defined* in terms of the new NumPy names, though (`-np.inf`, etc.). That would avoid any warnings/errors from upstream NumPy.

Reply:
Ah, you did that. Perfect.